demo/vl-wizard.html

```diff
@@ -0,0 +1,193 @@
+                            Volgende
+                        </button>
+                    </vl-wizard-pane>
+                    <vl-wizard-pane>
+                        <h3 is="vl-h3" slot="title">Stap 2</h3>
+                        <div is="vl-form-grid" is-stacked slot="content">
+                            <div is="vl-form-column" size="12">
+                                <label is="vl-form-label" for="years" block>Aantal jaren dienst</label>
+                                <input id="years" is='vl-input-field' block />
+                            </div>
+                        </div>
+                        <button is="vl-button-link" type="button" slot="previous-action">
+                            <span is="vl-icon" data-vl-icon="arrow-left-fat" data-vl-before></span>
+                            Vorige
+                        </button>
+                        <button is="vl-button" type="button" slot="next-action">
+                            Volgende
+                        </button>
+                    </vl-wizard-pane>
+                    <vl-wizard-pane>
+                        <h3 is="vl-h3" slot="title">Stap 3</h3>
+                        <div is="vl-form-grid" is-stacked slot="content">
+                            <div is="vl-form-column" size="12">
+                                <label is="vl-form-label" for="unavailable" block>Periode van
+                                    afwezigheid</label>
+                                <input id="unavailable" is='vl-input-field' block />
+                            </div>
+                        </div>
+                        <button is="vl-button-link" type="button" slot="previous-action">
+                            <span is="vl-icon" data-vl-icon="arrow-left-fat" data-vl-before></span>
+                            Vorige
+                        </button>
+                        <button is="vl-button" type="button" slot="next-action">
+                            Volgende
+                        </button>
+                    </vl-wizard-pane>
+                    <vl-wizard-pane>
+                        <h3 is="vl-h3" slot="title">Stap 4</h3>
+                        <div is="vl-form-grid" is-stacked slot="content">
+                            <div is="vl-form-column" size="12">
+                                <label is="vl-form-label" for="disease" block>Reden van ziekte</label>
+                                <input id="disease" is='vl-input-field' block />
+                            </div>
+                        </div>
+                        <button is="vl-button-link" type="button" slot="previous-action">
+                            <span is="vl-icon" data-vl-icon="arrow-left-fat" data-vl-before></span>
+                            Vorige
+                        </button>
+                        <button is="vl-button" type="button" slot="next-action">
+                            Versturen
+                        </button>
```

Comment:
ging ge nu niet voorzien om de wizard zelf de knoppen te laten renderen?

Reply:
nu moet ge ze toch elke keer nog zelf meegeven

Reply:
Dat is nog de uitbreiding die we moeten bespreken, zal dat via ander kanaal doen want ik had daar nog enkele vragen/bemerkingen.

Reply:
voor mij zijn dus de buttons eerder onderdeel van de wizard dan van elke afzonderlijke stap

Reply:
Issue #2 werd voorzien.